lib/cookbook-release/git-utilities.rb

```diff
@@ -7,7 +7,7 @@
 module CookbookRelease
   class GitUtilities
 
-    attr_accessor :no_prompt
+    attr_accessor :no_prompt, :sub_dir
```

Comment:
OPT: you allow to change the `sub_dir`, maybe an `attr_reader` would have been better? at least closer to original behavior

Reply:
Right